backend/btrixcloud/migrations/migration_0046_invalid_lang.py

```diff
@@ -0,0 +1,80 @@
+"""
+Migration 0046 - Invalid language codes
+"""
+
+from iso639 import is_language
+
+from btrixcloud.migrations import BaseMigration
+
+
+MIGRATION_VERSION = "0046"
+
+
+# pylint: disable=duplicate-code
+class Migration(BaseMigration):
+    """Migration class."""
+
+    # pylint: disable=unused-argument
+    def __init__(self, mdb, **kwargs):
+        super().__init__(mdb, migration_version=MIGRATION_VERSION)
+
+    async def migrate_up(self):
+        """Perform migration up.
+
+        Replace any invalid ISO-639-1 language codes that may be saved in
+        the database with "en".
+        """
+        configs_mdb = self.mdb["crawl_configs"]
+        orgs_mdb = self.mdb["organizations"]
+
+        # Fix workflows
+        async for config_raw in configs_mdb.find({"config.lang": {"$ne": None}}):
```

Comment:
Could we get the list of language codes and just match all with `$nin` - should be a bit faster I think.
Also need to do `crawls` as well, as they get a copy of the config from the workflow.